tensorflow_datasets/core/download/downloader.py

```diff
@@ -148,21 +149,39 @@ def _sync_download(self, url, destination_path):
         'https': os.environ.get('TFDS_HTTPS_PROXY', None),
         'ftp': os.environ.get('TFDS_FTP_PROXY', None)
     }
+    CA_BUNDLE = os.environ.get(
```

Comment:
Please add documentation on how this feature should be used (what things to set etc).

as I guess this is not only the matter of CA_BUNDLE, but user also needs to set the proxies, right ?

Reply:
Not really. Proxy is a different option. It is already been merged to the master branch. User might or might not set proxy. And also, user might or might not set the path to CA Bundle. Both are different things. And they depend on different environment variables. :)

Reply:
right CA_BUNDLE is only needed for SSL interception proxy:

```
export TFDS_HTTPS_PROXY="http://user:password@ip:port/"
export TFDS_CA_BUNDLE=path/ca_certs
```

Otherwise:

```
export TFDS_HTTPS_PROXY="http://user:password@ip:port/"
```
 is enough

I will validate if a full path or a local path is needed for the CA Certificate

Reply:
Right - even more important to have all these options documented.